sapi/fpm/fpm/fpm_stdio.c

```diff
@@ -130,10 +132,15 @@ static void fpm_stdio_child_said(struct fpm_event_s *ev, short which, void *arg)
 	} else {
 		event = &child->ev_stderr;
 	}
+	
+	/* calculate how long the "child 123 said into ..." prefix is so we can subtract its length from max_buf_size */
+	fmt_len = snprintf(fmt_buf, max_buf_size, "[pool %s] child %d said into %s: \"%%s\"%%s", child->wp->config->name, (int) child->pid, is_stdout ? "stdout" : "stderr");
+	/* subtract two characters each for the two "%s", then add character counts for "WARNING: " added by zlog and "[11-Feb-2015 15:58:20] " added by zlog; we're ignoring the potential extra length in log_level=DEBUG */
+	fmt_len = fmt_len - 2 - 2 + 9 + 23; 
```

Comment:
@Tyrael just reading this again and realised that it doesn't really makes much sense today as 1024 (that is specified in RFC3164) limit was obsoleted by RFC5424 where there is no such limit. So this reason should no longer be valid.